recipes/pcdsdevices/LICENSE.md

```diff
@@ -0,0 +1,39 @@
+Copyright (c) 2018, The Board of Trustees of the Leland Stanford Junior
```

Comment:
Can you delete this file and use the one from the source tarball?

Reply:
The latest release of the library does not yet include the license, though that will be fixed in the next release.